cmapPy/pandasGEXpress/diff_gctoo.py

```diff
@@ -0,0 +1,38 @@
+'''
+diff_gctoo.py
+
+Given a GCToo object calculates differential values (expression, viability etc.)
+'''
+import sys
+import cmapPy.math.robust_zscore as robust_zscore
+import cmapPy.pandasGEXpress.GCToo as GCToo
+
+def calc_differential(gctoo, plate_control=True, group_field='pert_type', group_val='ctl_vehicle', func = robust_zscore.calc_zscore):
```

Comment:
I like these args!

Reply:
For proteomics, I'll want to use this method exactly, but median normalization instead of z-scoring. A lot of your variable names are z-score centric. Could we change these to be something like `diff_gctoo` instead of `zscore_gctoo` and `diff_data` instead of `zscore_data`?